### ISSUE TITLE
Change the pixel art editor to something new and more supported

### ISSUE THREAD

**Issue by Oxey405** [4ian/GDevelop#4449]
Hello there ! So I'vhe been using Piskel for a LONG time now and I'm beginning to see where are it's limits.
It also started to crash on me without any prompt sometimes randomly when I do Ctrl+Z.

Maybe it's time for GDevelop to either switch to another pixel art editor or to make it's own (I wanted to make a pixel art editor myself so maybe it's time I start the project :))

As of now I think one of the best is Aseprite but it's paid so no wonders it won't be avaible for anyone to use on GDevelop but building a fork of Aseprite doesn't seems that complicated !

**Comment by Silver-Streak** [comment]
Unfortunately, aseprite is built on technology that isn't compatible, at all. It is c++, and not web compatible. Additionally, while it is source available, the license for the current version is not a permissive license. Even though you are welcome to download and compile the source for yourself, you are not allowed to distribute it, which would eliminate it from being bundled in GDevelop and be in compliance with their license.

Even libresprite, which was forked off the old 1.2 aseprite, is still incompatible technology.

Anything that would be bundled would need to be able to be bundled in javascript, typescript, React supported, etc, from the tech side.

License side it would need to be a fully permissive license like MIT.

Definitely agree that an alternative to Piskel qould be beneficial. There have been conversations before, but there's not really a lot of compatible libraries/applications out there currently.

**Comment by Oxey405** [comment]
> Unfortunately, aseprite is built on technology that isn't compatible, at all. It is c++, and not web compatible. Additionally, while it is source available, the license for the current version is not a permissive license. Even though you are welcome to download and compile the source for yourself, you are not allowed to distribute it, which would eliminate it from being bundled in GDevelop and be in compliance with their license.
> 
> Even libresprite, which was forked off the old 1.2 aseprite, is still incompatible technology.
> 
> Anything that would be bundled would need to be able to be bundled in javascript, typescript, React supported, etc, from the tech side.
> 
> License side it would need to be a fully permissive license like MIT.
> 
> Definitely agree that an alternative to Piskel qould be beneficial. There have been conversations before, but there's not really a lot of compatible libraries/applications out there currently.

Yes I am aware that aseprite cannot be integrated I was more taking it as an example :)
But I'm really sure it's a better idea to switch from Piskel to something else because it's simply too old and doesn't have enough features for "true" pixel art
